.circleci/Dockerfile

```diff
@@ -0,0 +1,9 @@
+# Upgrade the helm client version here when the az cluster tiller version
+# upgrades.
+FROM lachlanevenson/k8s-helm:v2.13.1
+
+FROM lachlanevenson/k8s-kubectl:v1.14.1
+
+FROM microsoft/azure-cli:latest
```

Comment:
I was thinking the same. According to their service principal docs for auth, we should use
>the Azure CLI version 2.0.61 or later
but their tagged versions stop before this.

Initially I wrote an alpine 3.9 Dockerfile for the azure-cli, but was trying to stay minimal with what we maintain. Now that you mention it, maybe I should go back to this (there are [some](https://github.com/Azure/azure-cli/issues/8863) [issues](https://github.com/Azure/azure-cli/issues/9167) that solve for this which I referred to). This will likely fix those CVEs.

Reply:
Image tagged, CVEs [fixed](https://quay.io/repository/helmpack/kubectl-helm-az/manifest/sha256:f880d3797da1c914a7bedf2674741d6b2ad2b07c0c7c8caa16efcb15f3d49a96?tab=vulnerabilities) 👍